Dockerfile

```diff
@@ -14,7 +14,12 @@ RUN apt-get update \
         sudo \
         unzip \
         zip \
-    && curl -sL https://deb.nodesource.com/setup_10.x | bash - \
+    && apt-get update \
```

Comment:
Given that there's an `apt-get update` on line 9 and no changes to a source, don't need to do this line here and can remove.